controllers/create_clustermanagementaddon.go

```diff
@@ -50,4 +51,9 @@ func newClusterManagementAddOn(instance *searchv1alpha1.Search) *addonv1alpha1.C
 			},
 		},
 	}
+	err := controllerutil.SetControllerReference(instance, cma, r.Scheme)
+	if err != nil {
+		log.V(2).Info("Could not set control for search-collector ClusterManagementAddOn", err)
```

Comment:
Do we need to return error and retry here?

Reply:
No a serious error , we dont want to retry here .